alb/terraform-examples/nsxt-cloud-data-segment/Create Subsequent Data Segments

```diff
@@ -0,0 +1,179 @@
+1. create tf script:
```

Comment:
Should this be a markdown file with incline HCL code blocks or should it be a set of .tf files?